server/src/controllers/orderController.ts

```diff
@@ -187,6 +222,65 @@ orderController.get('/', async (req: Request, res: Response) => {
   }
 });
 
+
+orderController.get('/active', async (req: Request, res: Response) => {
```

Comment:
We may not want to use `active` since we want to protect the meaning of active to be for events and seasons only. Might be better to use something else

Reply:
Sounds good, I ended up writing a different implementation of the route on branch 95 (refund backend). So we can just delete this route from the file if we want to.

Reply:
Ok, lets delete this then

Reply:
Fixed